src/backend/dx12/src/native.rs

```diff
@@ -256,7 +257,8 @@ pub struct DescriptorPool {
     pub pools: Vec<pso::DescriptorRangeDesc>,
     pub max_size: u64,
 }
-unsafe impl Send for DescriptorPool { }
+unsafe impl Send for DescriptorPool {}
+unsafe impl Sync for DescriptorPool {}
```

Comment:
Why derive `Sync` here? Do you need it?

Reply:
I don't think this is correct :( but yes I happen to need this at the moment.
Guess it's a good case for branching out then